staging/src/k8s.io/client-go/util/consistencydetector/watch_list_data_consistency_detector.go

```diff
@@ -36,19 +32,3 @@ func init() {
 func IsDataConsistencyDetectionForWatchListEnabled() bool {
```

Comment:
Is this still used anywhere else?

Reply:
Should we move this to that file then?

Reply:
I somehow missed the core of that package - I would actually live it where it is now.

But maybe what I would do is to move the remainder of this file directly into data_consistency_detector.go here just for completeness.